stable/etcd-operator/templates/cluster.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if .Values.cluster.enabled -}}
-apiVersion: "coreos.com/v1"
-kind: "EtcdCluster"
+{{- if and .Values.cluster.enabled (not .Release.IsInstall) -}}
```

Comment:
Since the version is below 1.0.0, breaking changes may happen on minor release versions according to semver. 

http://semver.org/#spec-item-4